src/libs/OptimisticReportNames.ts

```diff
@@ -210,6 +234,7 @@ function computeReportNameIfNeeded(report: Report | undefined, incomingUpdate: O
     const formulaContext: FormulaContext = {
         report: updatedReport,
         policy: updatedPolicy,
+        updatedTransaction,
```

Comment:
Can we please get the updatedTransaction from the update like we do for updatedReport, updatedPolicy?